--- pce-0.1.5/src/arch/sim6502/Makefile.in~	2004-08-02 23:05:01.000000000 +0200
+++ pce-0.1.5/src/arch/sim6502/Makefile.in	2004-08-03 19:29:33.000000000 +0200
@@ -64,8 +64,8 @@
 
 
 install: install-bin sim6502.cfg
-	$(INSTALL) -d -m 755 $(etcdir)
-	test -f $(etcdir)/sim6502.cfg || $(INSTALL) -m 644 sim6502.cfg $(etcdir)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(etcdir)
+	test -f $(etcdir)/sim6502.cfg || $(INSTALL) -m 644 sim6502.cfg $(DESTDIR)$(etcdir)
 
 
 sim6502: $(OBJ) $(OBJ_EXT)
